app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreenState.kt

```diff
@@ -68,8 +68,11 @@ class ConversationScreenState(
         coroutineScope.launch { modalBottomSheetState.animateTo(ModalBottomSheetValue.Expanded) }
     }
 
-    fun hideEditContextMenu() {
-        coroutineScope.launch { modalBottomSheetState.animateTo(ModalBottomSheetValue.Hidden) }
+    fun hideEditContextMenu(onComplete: () -> Unit) {
+        coroutineScope.launch {
+            modalBottomSheetState.animateTo(ModalBottomSheetValue.Hidden)
+            onComplete()
```

Comment:
Making sure we allow user to execute whatever is needed only after the animation is done, otherwise it will throw `CancellationException`